pkg/ingress/status.go

```diff
@@ -55,26 +55,32 @@ func recordStatus(at interface{}, reason string, err error, status v1.ConditionS
 			v.Status.Conditions = &conditions
 		}
 		meta.SetStatusCondition(v.Status.Conditions, condition)
-		_, _ = kube.GetApisixClient().ApisixV1().ApisixTlses(v.Namespace).
-			UpdateStatus(context.TODO(), v, metav1.UpdateOptions{})
+		if _, errRecord := kube.GetApisixClient().ApisixV1().ApisixTlses(v.Namespace).
+			UpdateStatus(context.TODO(), v, metav1.UpdateOptions{}); errRecord != nil {
+			log.Errorf("resource record error: %s/%s, with err %s", v.Namespace, v.Name, errRecord.Error())
```

Comment:
Your error log lost some vital contexts like recording which resource? What does the resource look like?

```suggestion
			log.Errorw("failed to record status change for ApisixTls",
			        zap.Error(err),
			        zap.String("name", v.Name),
			        zap.String("namespace", v.Namespace),
			)
```

Reply:
Done